specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-08-01-preview/IncidentCreationRules.json

```diff
@@ -0,0 +1,436 @@
+        "operationId": "IncidentCreationRules_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/IncidentCreationRuleId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IncidentCreationRules_Delete": {
+            "$ref": "./examples/incidentCreationRules/IncidentCreationRules_Delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "GroupingKind": {
+      "enum": [
+        "PropertyBased"
+      ],
+      "type": "string",
+      "example": "PropertyBased",
+      "x-ms-enum": {
+        "name": "GroupingKind",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "PropertyBased"
+          }
+        ]
+      }
+    },
+    "IncidentCreationRule": {
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/2.0/types.json#/definitions/ResourceWithEtag"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "$ref": "#/definitions/IncidentCreationRuleProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "IncidentCreationRuleCondition": {
+      "description": "Describes a incident creation rule condition.",
+      "type": "object",
+      "properties": {
+        "conditionType": {
+          "$ref": "#/definitions/IncidentCreationRuleConditionType"
+        }
+      }
+    },
+    "IncidentCreationRuleConditionType": {
+      "enum": [
+        "Property"
+      ],
+      "type": "string",
+      "example": "Property",
+      "x-ms-enum": {
+        "name": "IncidentCreationRuleConditionType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Property",
+            "description": "Evaluate an object property value."
+          }
+        ]
+      }
+    },
+    "IncidentCreationRuleGroupingConfiguration": {
```

Comment:
>IncidentCreationRuleGroupingConfiguration [](http://example.com/codeflow?start=5&length=41)

groupconfig seems to have a lot of properties in the example that aren't in the spec as well

Reply:
added